### DISCUSSION TITLE
How to identify a click outside the Canvas frame?

### DISCUSSION THREAD

**Discussion by ivent** [fabricjs/fabric.js#8003] [Q&A] [answered]
I'm creating a project in Fabricjs Canvas and I would like to know how to identify a click outside the canvas area, I want to deselect all objects with the canvas.discardActiveObject().renderAll() function; however it only works when the click occurs within the canvas, I need to activate it when I click outside, I even created a function to activate this function in the div that the canvas is inside but when I click on the canvas it is also activated, could someone tell me help I've looked everywhere and haven't found a solution that should be simple.

**Comment by ShaMan123** [comment, marked as answer]
This is unrelated to fabric.
It's a HTML how-to
listen to the document/window
